test/storage/index.js

```diff
@@ -192,8 +192,4 @@ describe('Storage', function() {
       });
     });
   });
-
-  describe('makeReq_', function() {
```

Comment:
Sorry, hard to tell given that I commented on the `describe` line but it's an empty `describe` block with no tests in it. I'm just removing it because it's useless. Not sure what happened to those tests if they ever existed in the first place.